packages/bar/src/BarCanvas.tsx

```diff
@@ -285,6 +311,8 @@ const InnerBarCanvas = <RawDatum extends BarDatum>({
         ]
     )
 
+    const formatValue = useValueFormatter(valueFormat)
```

Comment:
Rather than duplicating the formatting logic, I think it should be part of the hook computing the totals, which would provide an extra `formattedValue`.